libtcmu_config.c

```diff
@@ -459,11 +465,11 @@ struct tcmu_config *tcmu_setup_config(const char *path)
 	}
 
 	/*
-	 * f the dynamic reloading thread fails to start, it will fall
+	 * If the dynamic reloading thread fails to start, it will fall
 	 * back to static config
 	 */
 	if (pthread_create(&cfg->thread_id, NULL, dyn_config_start, cfg)) {
-		tcmu_warn("Failed to start the dynamic config reloading feature!\n");
+		tcmu_warn("Dyn config failed and falls back to static!\n");
```

Comment:
Over all looks good to me.

Just had this suggestion in mind,
how about, tcmu_warn("dynamic config (start) failed, falling back to static.\n");
or something similar ?